chapters/en/slides/module3/module3_01.Rmd

```diff
@@ -0,0 +1,274 @@
+
+```{python}
+alt.Chart(movies[:50]).mark_point().encode(
+    x='runtime')
+```
+
+Notes:
+When there are more observations,
+it becomes really hard to see which ones are where
+since the circular marks are overlapping with each other.
+
+When it becomes hard to tell individual observations apart,
+we say that a plot become saturated.
+We will talk more about this concept later in the module.
+
+---
+
+## Thinner marks avoids saturation
+
+```{python}
+alt.Chart(movies[:50]).mark_tick().encode(
+    x='runtime')
+```
+
+Notes:
+It is often easier to separate individual observations
+when they are represented as thin tick marks instead of the thick points
+as in the previous slide.
+
+The disadvantage of this plot type
+is that it can be a bit harder to see individual observations.
+However,
+when we are visualization many marks,
+we are often more interested in how the data is shaped (or distributed) in general,
+rather than looking at individual points.
+
+This type of plot with vertical tick marks along one axis
+is so common that it is sometimes referred to by a special name: rugplot.
+
+The visualization grammar of Altair let's us think about it more intuitively
+by composing it with a mark and a single encoding,
+instead of remembering a specific name.
+
+---
+
+## Thinner marks also saturate for large data sets
+
+```{python}
+alt.Chart(movies).mark_tick().encode(
+    x='runtime')
+```
+
+Notes:
+However,
+when we increase the number of observations even further (to about 500)
+it becomes difficult to tell exactly how the data distribution looks,
+even with these thin marks.
+
+---
+
+## Using histograms to visualize distributions avoids saturation issues
+
+```{python}
+alt.Chart(movies).mark_bar().encode(
+    alt.X('runtime', bin=alt.Bin(maxbins=30)),
+    y='count()')
+```
+
+Notes:
+Instead of visualizing each individual observation,
+we can use the histograms we learnt about in last module,
+where we divide one axis into groups (or "bins")
+and represent the count of observations in each group with bar.
+
+In this plot,
+we can directly see that most movies are around 100 min in length.
+But from our experience,
+we know that factors such as genre and production year,
+can influence movie length.
+
+How can we separate the data based on other columns in the dataset
+and compare multiple distributions?
+
+---
+
+## Comparing multiple distributions via faceting
+
+```{python}
+(alt.Chart(movies).mark_bar().encode(
+    alt.X('runtime', bin=alt.Bin(maxbins=30)),
+    y='count()')
+ .facet('country'))
+```
+
+Notes:
+Let's start by comparing the movie length broken down per country.
+Here we use faceting which we learnt about in the last module,
+to create one histogram per country.
+
+You can see that this dataset only contains movies made in the UK or the US.
+It does seem like the distribution of movies is pretty equal between the two countries,
+but it is a bit hard to see because there are so many more movies made in the US.
```

Comment:
I don't know if I agree. The shape looks flatter (because of the binning), so maybe I would say here that the range and the centre looks similar, but maybe the shape is different, its hard to tell because there are so many more movies in the US...

Reply:
Yes, the main point I want to get across that it is hard to tell and that we can scale them to make it easier to compare